tests/entrypoints/test_openai_server.py

```diff
@@ -198,15 +201,19 @@ async def test_completion_streaming(server, client: openai.AsyncOpenAI,
     single_output = single_completion.choices[0].text
     single_usage = single_completion.usage
 
-    stream = await client.completions.create(
-        model=model_name,
-        prompt=prompt,
-        max_tokens=5,
-        temperature=0.0,
-        stream=True,
-    )
+    stream = await client.completions.create(model=model_name,
+                                             prompt=prompt,
+                                             max_tokens=5,
+                                             temperature=0.0,
+                                             stream=True,
+                                             logprobs=True,
+                                             top_logprobs=10)
```

Comment:
`top_logprobs` doesn't exist in that api. Did you pass this test on your machine? 🤔

Reply:
Ah sorry. fixed!